bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -130,12 +130,16 @@ func (r *KubeadmConfigReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, re
 
 	// Look up the owner of this KubeConfig if there is one
 	configOwner, err := bsutil.GetConfigOwner(ctx, r.Client, config.ObjectMeta)
+	if apierrors.IsNotFound(err) {
+		// Could not find the owning machine yet, this is actually ok and we just need to wait a second
+		return ctrl.Result{}, nil
+	}
 	if err != nil {
-		log.Error(err, "could not get owner")
+		log.Error(err, "failed to get owner")
 		return ctrl.Result{}, err
 	}
 	if configOwner == nil {
-		log.Info("Waiting for the configuration owner's controller to set OwnerRef on the KubeadmConfig")
+		log.V(5).Info("Waiting for the configuration owner's controller to set OwnerRef on the KubeadmConfig")
```

Comment:
```suggestion
		log.V(3).Info("Waiting for the configuration owner's controller to set OwnerRef on the KubeadmConfig")
```

Never seen high as 5 before, maybe we can just remove it if it's not useful?

Reply:
That's fine with me. My logic here was we tend to default to 4 and I don't really want to see this log any more. Will remove